Sources/Models/Views/StackView.swift

```diff
@@ -95,11 +88,9 @@ public struct StackView: IBDecodable, ViewProtocol, IBIdentifiable {
             variations:                                variationContainer.elementsIfPresent(of: .variation),
             backgroundColor:                           colorsContainer?.withAttributeElement(.key, CodingKeys.backgroundColor.stringValue),
             tintColor:                                 colorsContainer?.withAttributeElement(.key, CodingKeys.tintColor.stringValue),
-            axis:                                      container.attributeIfPresent(of: .axis) ?? "horizontal",
             distribution:                              container.attributeIfPresent(of: .distribution),
-            alignment:                                 container.attributeIfPresent(of: .alignment),
             spacing:                                   container.attributeIfPresent(of: .spacing),
-            baselineRelativeArrangement:               container.attributeIfPresent(of: .baselineRelativeArrangement)
```

Comment:
Same here removed, must not. Please check your entire pr if you remove other attributes